vertical-pod-autoscaler/pkg/common/admission_controller_status.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I'd rather see this in pkg/utils/status

Reply:
Done. Consts in pgk/utils/status/status_object.go.